membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -97,14 +97,13 @@ class AttributeController extends Controller with LoggingWithLogstashFields {
       import loghandling.StopWatch
       val stopWatch = new StopWatch
 
-      futureResult.map { result: Disjunction[String, R] =>
-        val latency = stopWatch.elapsed
-        val customFields: List[LogField] = List("zuora_latency_millis" -> latency.toInt, "zuora_call" -> whichCall, "identityId" -> identityId)
-        result match {
-          case -\/(messageOrError) => {
-            logWarnWithCustomFields(s"$whichCall failed with ${messageOrError}", customFields)
-          }
-          case \/-(_) => logInfoWithCustomFields(s"$whichCall took ${latency}ms", customFields)
+      futureResult.map { disjunction: Disjunction[String, R] =>
+        disjunction match {
+          case -\/(message) => log.info(message)
```

Comment:
perhaps better to make this warn with some further explanation attached
`log.warn(s"$whichCall failed with ${message}")`